### ISSUE TITLE
Receiving Private Messages with Screen Reader

### ISSUE THREAD

**Issue by LizHareDogs** [thelounge/thelounge#4199]
### Feature Description
Hi, I am using a screen reader to access TheLounge (Mac, VoiceOver, Chrome). 

Using the screen reader allows us to navigate the elements in the browser linearly. I've been sent private messages, And I'm unable to locate a way to read them. I've tried the tab with my username, but that doesn't provide anything useful. 

We have a feature where a bot can send us information about our event schedule, but when I request it, I can't find where it goes.

Thank you,
Liz
